delivery/app.py

```diff
@@ -6,6 +6,44 @@
 from delivery.error_handling import CustomError, WrongIdError
 from flasgger import Swagger
 from marshmallow import Schema, fields, ValidationError
+from abc import ABC, abstractmethod
+
+
+class AbstractRestaurantRepository(ABC):
```

Comment:
`app.py` is quite big already, try to create several files like `repositories.py`, `endpoints.py`, `schemas.py`

Reply:
fixed